### PR TITLE
Updated the link to the test-suit

### PR DESCRIPTION
**Pull request by yatri1609**
This PR updates the "test-suite" link at the top of the spec to point to the new test suite location. 

This is in context with #54

***
<a href="https://pr-preview.s3.amazonaws.com/yatri1609/payment-method-id/pull/55.html" title="Last updated on Sep 30, 2018, 4:58 AM GMT (fc31c71)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-method-id/55/14f2321...yatri1609:fc31c71.html" title="Last updated on Sep 30, 2018, 4:58 AM GMT (fc31c71)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
       specStatus: "ED",
       // We can only indirectly test conformance, so we rely on certain tests
       // in the Payment Request API.
-      testSuiteURI: "https://w3c-test.org/payment-request/",
+      testSuiteURI: "https://w3c-test.org/payment-method-id/",
```
Comment (marcoscaceres) [inline index.html:38]: see my comment in the related bug about adding the implementation report property too.

**Comment by marcoscaceres** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by yatri1609** [comment]
@marcoscaceres I just did the change

**Review by marcoscaceres — APPROVED** [review]
Looks great! Ok, let's get the actual report in place at: https://w3c.github.io/test-results/payment-method-id/all.html

**Comment by marcoscaceres** [comment]
Will merge after the report is in place.

**Comment by yatri1609** [comment]
@marcoscaceres Is it fine now? As I'm not getting what are you actually trying to say

**Comment by marcoscaceres** [comment]
@yatri1609 sorry, let me try agin. The files we placed in:

https://github.com/w3c/test-results/tree/gh-pages/payment-request

Are in the wrong place, they should be in a directory called "payment-method-id" instead.

**Comment by yatri1609** [comment]
Since there is no such directory so I'll make one for it and update it. Sorry I got a bit confused

**Comment by marcoscaceres** [comment]
>  Sorry I got a bit confused

Oh no problem. I also didn't spot it during review.

**Comment by yatri1609** [comment]
@marcoscaceres Can you please have a look at it again?

**Pull request merged**